### PR TITLE
Creation of a user model

### PR DESCRIPTION
**Pull request by NissZer**

### CODE REVIEW

File: todo/todo/models.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    user_name = models.CharField(max_length=10)
+    password = models.CharField(max_length=10)
```
Comment (ayoubkachkach) [inline todo/todo/models.py]: Setting max_length to 10 sounds arbitrary. ayoubkachkach (github username) is 13 characters long. I'd suggest you double the length.

File: todo/todo/models.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+# Create your models here.
```
Comment (ayoubkachkach) [inline todo/todo/models.py]: delete.

File: todo/todo/models.py (hunk 1 above)
Comment (NissZer) [reply todo/todo/models.py]: okay mate.

File: todo/todo/models.py (hunk 2 above)
Comment (NissZer) [reply todo/todo/models.py]: the comment?

Comment (ayoubkachkach) [reply todo/todo/models.py]: yes

**Pull request merged**